packages/nestjs/src/setup.ts

```diff
@@ -78,6 +78,30 @@ class SentryGlobalFilter extends BaseExceptionFilter {
 Catch()(SentryGlobalFilter);
 export { SentryGlobalFilter };
 
+/**
+ * Global filter to handle exceptions and report them to Sentry.
+ *
+ * The BaseExceptionFilter does not work well in GraphQL applications.
+ * By default, Nest GraphQL applications use the ExternalExceptionFilter, which just rethrows the error:
+ * https://github.com/nestjs/nest/blob/master/packages/core/exceptions/external-exception-filter.ts
+ *
+ * The ExternalExceptinFilter is not exported, so we reimplement this filter here.
+ */
+class SentryGlobalGraphQLFilter {
+  public static readonly __SENTRY_INTERNAL__ = true;
```

Comment:
I just noticed this but let's remove the static keyword from this field. Also in all the other places where we have it. We are not using it in a static way and defining a static in JS is considered a side-effect and that becomes problematic in some scenarios. (it inhibits tree shaking for example)